File: flutter/fctconnect/lib/models/sala_get_data.dart
class SalaGetData {
  final String id;
  final String name;
  final String building;
  final double lat;
  final double lng;
  final int capacity;
 
  SalaGetData({
    required this.id,
    required this.name,
    required this.building,
    required this.lat,
    required this.lng,
    required this.capacity,
  });

  factory SalaGetData.fromJson(Map<String, dynamic> json) {
    return SalaGetData(
      id: json['id'],
      name: json['name'],
      building: json['building'],
      lat: json['lat'],
      lng: json['lng'],  
      //url: json['url'],
      capacity: json['capacity'],
    );
  }
}

File: java&html/src/main/java/pt/unl/fct/di/apdc/firstwebapp/util/SalaGetData.java
package pt.unl.fct.di.apdc.firstwebapp.util;

import java.util.List;

public class SalaGetData {

    private String name;
    private String building;
    private String url;
    private String id;
    //private String qrCodeUrl;
    private List<Integer> lotation;
    private double lat;
    private double lng;
    private long capacity;

    public SalaGetData(){}

    public SalaGetData(String name, String building, String url, String id,
                        List<Integer> lotation, double lat, double lng, long capacity) {
        this.name = name;
        this.building = building;
        this.url = url;
        this.id = id;
        this.lotation = lotation;
        this.lat = lat;
        this.lng = lng;
        this.capacity = capacity;
    }

    public String getName() {
        return name;
    }

    public String getBuilding() {
        return building;
    }

    public String getUrl() {
        return url;
    }

    public String getId() {
        return id;
    }

    public List<Integer> getLotation(){
        return lotation;
    }

    public double getLat() {
    	return lat;
    }

    public double getLng() {
    	return lng;
    }

    public long getCapacity() {
        return capacity; 
    }
}